build/images/base-windows/Dockerfile

```diff
@@ -61,3 +61,4 @@ COPY --from=windows-utility-base ["C:\\\\wins","C:\\\\wins"]
 COPY --from=windows-golang ["C:\\\\Program Files\\\\Go", "C:\\\\Program Files\\\\Go"]
 
 RUN setx /m PATH "C:\Program Files\7-Zip;C:\git\bin;C:\git\usr\bin;C:\mingw64\bin;C:\Program Files\Go\bin;%GOPATH\bin%;%PATH%"
+
```

Comment:
Should we import the ovs image into base image dockerfile?

Reply:
windows base image might not need to have ovs as it might increase the size of base image as well as we specifically do not have any use case for it. Also the ovs image works independently and do not need other dependencies like go and others, which are available in base image.

Reply:
if I understand correctly, you want to import the OVS image into the agent image Dockerfile, is that correct?

Reply:
agent uses base-windows image based on agent image dockerfile and antrea-ovs will use windows-ovs image , we have now a separate dockerfile for building ovs image, so we will not import ovs in agent dockerfile

Reply:
I don't think it is necessary to maintain two images on Windows seperately: one for antrea-agent and the other for ovs. On Linux, we also package both antrea-agent and antrea-ovs into the a single image.

Reply:
I would prefer only maintaining one agent image that can be used for both OVS and the agent container. Do you have any other suggestions? @tnqn @wenyingd

Reply:
@wenyingd , currently we have 3 windows images windows-utility , windows go & windows base image(contains windows utlity and windows go), Now we have another image built which is windows-ovs which is not imported in windows base image currently. @XinShuYang although we can prefer one windows base image, However antrea ovs might not need everything which base windows image has, if we want to keep the image lighter (Usually windows-ovs image is 4gb and windows base image is ~5-6gb size)